spec/lib/addic7ed/episode_spec.rb

```diff
@@ -108,34 +116,34 @@
 
     it 'gets the best subtitle candidate with a network request' do
       expect(episode).to receive(:best_subtitle).once.and_call_original
-      episode.download_best_subtitle!('fr')
+      episode.download_best_subtitle!('fr', false)
```

Comment:
The second argument here (`false`) can now be removed :smiley: 
Same thing at lines 126, 133, 139, 146, 156 and 162.